x-pack/plugins/ingest_manager/common/services/decode_cloud_id.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// decodeCloudId decodes the c.id into c.esURL and c.kibURL
+export function decodeCloudId(
```

Comment:
Not blocking but I think it would be nice to have a test for that file and I think you already have it in you TS playground.

Reply:
Thanks for your help here. Added in https://github.com/elastic/kibana/pull/65366/commits/cf7b45a3faae6558c763b4f6a4fa0a78eca3afd9